src/utils/config.ts

```diff
@@ -0,0 +1,31 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import * as dotenv from 'dotenv';
+import { ConfigMaestro } from '../../types/global';
+
+dotenv.config();
+
+const configFilePath = process.env.CONFIG_COE_PATH || path.resolve('./config.json');
```

Comment:
This is always in the `resources/` directory of the bundled application in executable form. Why is it needed in the config file?

Reply:
This loads the configuration file needed to run the cosimulation, not maestro itself: without configuring a path, the remaining option would be asking the user for the path.